lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -1018,6 +1019,102 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
             , expectField #expiresAt (`shouldSatisfy` isJust)
             ]
 
+    it "TRANS_MINT_01 - Mint tokens" $ \ctx -> runResourceT $ do
+      w <- fixtureWallet ctx
+
+      addrs <- listAddresses @n ctx w
+      let destination = (addrs !! 1) ^. #id
+      let payload = Json [json|{
+                              "mint_burn": {
+                                  "monetary_policy_index": "0",
+                                  "token_name": "aaaa",
```

Comment:
Perhaps it would be better to have `asset_name` for consistency? I believe we are rather using `asset` instead of `token` in all other places, i.e.  https://input-output-hk.github.io/cardano-wallet/api/edge/#tag/Assets